eureka2-read-server/src/main/java/com/netflix/eureka2/server/FullFetchInterestClientProvider.java

```diff
@@ -22,12 +22,15 @@
 import com.netflix.eureka2.server.config.EurekaCommonConfig;
 import com.netflix.eureka2.server.interest.FullFetchBatchingRegistry;
 import com.netflix.eureka2.server.interest.FullFetchInterestClient;
+import com.netflix.eureka2.transport.TransportClient;
 
 /**
  * @author Tomasz Bak
  */
 public class FullFetchInterestClientProvider implements Provider<FullFetchInterestClient> {
 
+    private static final String EUREKA_READ_CLIENT_ID = "eurekaReadClient";
```

Comment:
How about change all the default client names to something like client(InterestClient, FullFetchClient).class.getSimpleName()? That way they can be nice and programmatic.